gulpfile.js

```diff
@@ -491,10 +491,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.19.1'))
+    .pipe(replace('[ios.current-app-version]', '2.20.0'))
```

Comment:
```suggestion
    .pipe(replace('[ios.current-app-version]', '2.20.1'))
```

Will probably be 2.20.1, see latest RC: https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.20.0-rc.5

Reply:
thanks again @Ein-Tim